### DISCUSSION TITLE
IPv6 support

### DISCUSSION THREAD

**Discussion by imp1sh** [tbnobody/OpenDTU#1214] [General]
Hi,

thank you for that wonderful project. I'm really glad I don't have to rely on that cloud crap.
Would you kindly suggest implementing IPv6 at best also with DHCPv6 client support.

That is all.

**Comment by imp1sh** [comment]
@tbnobody what do you think?

**Comment by ds2k5** [comment]
Hi @ALL 
is there a plan to add IPV6 to the next Version ?
